pkgs/by-name/cr/cruft/package.nix

```diff
@@ -1,34 +1,35 @@
 {
   python3Packages,
-  fetchFromGitHub,
+  fetchPypi,
   lib,
 }:
 python3Packages.buildPythonApplication rec {
   pname = "cruft";
-  version = "2.15.0";
+  version = "2.16.0";
   pyproject = true;
 
-  src = fetchFromGitHub {
-    owner = "cruft";
-    repo = "cruft";
-    rev = version;
-    hash = "sha256-qIVyNMoI3LsoOV/6XPa60Y1vTRvkezesF7wF9WVSLGk=";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-GEZihT+8HBbAE3/g6URODL6V5ANi9euy2frjPZIj5z8=";
   };
 
   build-system = with python3Packages; [
-    poetry-core
+    hatchling
   ];
 
   nativeCheckInputs = with python3Packages; [
     pytest7CheckHook
```

Comment:
Can we use the regular `pytestCheckHook`?

Reply:
No, upstream uses `pytest.raises(None)` and passing None is deprecated, it raises an error in v8